nautobot/core/views/utils.py

```diff
@@ -98,8 +98,14 @@ def csv_format(data):
     return ",".join(csv)
 
 
-def get_obj_from_context(context):
-    """From the given context, return the object that is in the context"""
+def get_obj_from_context(context, key=None):
+    """From the given context, return the `object` that is in the context.
+
+    If a key is specified, return the value for that key.
+    Otherwise return the value for either of the keys `"obj"` or `"object"` as default behavior.
+    """
+    if key is not None:
+        return context.get(key)
     return context.get("obj") or context.get("object")
```

Comment:
Coud this be `context.get(key) or context.get("obj") or context.get("object")` or will it throw an error when `key` is None?